### PR TITLE
1.0.180

### PR DESCRIPTION
**Pull request by jshuman1**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/mcafee-base.template [hunk 1]
```diff
@@ -108,10 +108,12 @@ Parameters:
     Type: AWS::EC2::VPC::Id
   PublicSubnet1ID:
     Description: ID of public subnet 1 in Availability Zone 1 for the ELB load balancer (e.g., subnet-9bc642ac)
-    Type: AWS::EC2::Subnet::Id
```
Comment (sshvans) [inline templates/mcafee-base.template]: What is the reason to change the type from AWS intrinsic type to string?

File: templates/mcafee-base.template [hunk 2]
```diff
@@ -120,7 +122,8 @@ Parameters:
     Type: AWS::EC2::Subnet::Id
   BastionSecurityGroupID:
     Description: ID of the bastion host security group to enable SSH connections (e.g., sg-7f16e910)
-    Type: AWS::EC2::SecurityGroup::Id
```
Comment (sshvans) [inline templates/mcafee-base.template]: Same question as above?

File: templates/mcafee-base.template (hunk 1 above)
Comment (vvanjara) [reply templates/mcafee-base.template]: We started supporting ePO deployment only in private subnet (based on some customer requirements) and public subnet inputs are optional in that scenario.
Input parameter is mandatory if we set the intrinsic type for these two input parameters so we changed the type as string and take some decisions inside the template based on this parameter value.

File: templates/mcafee-base.template [hunk 3]
```diff
@@ -108,10 +108,12 @@ Parameters:
     Type: AWS::EC2::VPC::Id
   PublicSubnet1ID:
     Description: ID of public subnet 1 in Availability Zone 1 for the ELB load balancer (e.g., subnet-9bc642ac)
-    Type: AWS::EC2::Subnet::Id
+    Default: ''
+    Type: String
   PublicSubnet2ID:
     Description: ID of public subnet 2 in Availability Zone 2 for the ELB load balancer (e.g., subnet-e3246d8e)
-    Type: AWS::EC2::Subnet::Id
```
[the remark this answers is not in the document]
Comment (vvanjara) [reply templates/mcafee-base.template]: We started supporting ePO deployment only in private subnet (based on some customer requirements) and public subnet inputs are optional in that scenario.
Input parameter is mandatory if we set the intrinsic type for these two input parameters so we changed the type as string and take some decisions inside the template based on this parameter value.

File: templates/mcafee-base.template (hunk 2 above)
Comment (vvanjara) [reply templates/mcafee-base.template]: We started supporting ePO deployment only in private subnet (based on some customer requirements) and public subnet inputs are optional in that scenario.
Input parameter is mandatory if we set the intrinsic type for these two input parameters so we changed the type as string and take some decisions inside the template based on this parameter value.

File: templates/mcafee-base.template (hunk 1 above)
Comment (sshvans) [reply templates/mcafee-base.template]: Sounds good. Thanks!

**Pull request merged**